figures/create-endpoint-activity.mmd

```diff
@@ -0,0 +1,21 @@
+stateDiagram-v2
+
+    state "Client hits /create endpoint" as conn 
+    state "Client tries to edit room parameters" as param
+    state fork_state <<fork>>
+    state "Title greater than 255 char" as title
+    state "Name is less than 3 char or greater than 32 char" as name
+    state "Visibility is not an allowed value" as visibility
+    state join_state <<fork>>
+    [*] --> conn: Client connects.
+    conn --> param: 
```

Comment:
The parameters are already present on the request when we receive it. This should be one step.